test/NewsService.Data.UnitTests/NewsRepositoryTests.cs

```diff
@@ -18,7 +20,9 @@ class NewsRepositoryTests
     {
         private IDataProvider _provider;
         private INewsRepository _repository;
+        private Mock<IHttpContextAccessor> _accessorMock;
```

Comment:
better rename _accessorMock to _httpContext

Reply:
done